tsconfig-serve.json

```diff
@@ -1,25 +1,23 @@
 {
   "compilerOptions": {
-    "sourceMap": true,
     "declaration": false,
-    "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "lib": [
+      "es2020",
```

Comment:
@cal2195 -- if you happen to know what's going on here, I'd appreciate a comment -- I have almost no idea what I'm doing 😅 

I was thinking of updating things to latest libraries -- and everything seems to work, but the one that messed me up is the `target` (see line 12) -- "es5" -- when I bumped it up to `ESNext` or `ES2020` everything failed (`main.ts` errored out on the first line with an error, something like "can't use Import outside of module" 🤷 

I would have thought these things would be backward compatible ... though I may be thinking wrong about everything 🤷 I updated to Node 14 and it still didn't help -- so I'm unsure what's happening.

If you happen to know the about `target` thing and can comment, please. Otherwise, no worries -- everything is working fine 👌